lib/local-links-manager/check_links/link_status_requester.rb

```diff
@@ -24,11 +24,11 @@ def check_authority_urls(authority_slug)
 
       def urls_for_authority(authority_slug)
         local_authority = LocalAuthority.find_by(slug: authority_slug)
-        local_authority.links.map(&:url) << local_authority.homepage_url
+        local_authority.links.with_url.map(&:url) << local_authority.homepage_url
       end
 
       def homepage_urls
-        LocalAuthority.all.map(&:homepage_url)
+        LocalAuthority.all.map(&:homepage_url).sort
```

Comment:
Does the link checker require them sorted?

Reply:
It doesn't but I wanted to have them in a deterministic order, partly for my OCD and partly so the test didn't randomly fail.  I could have been a little more overt about this 🍇